modules/registry-configuring-storage-openstack-user-infra.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring-registry-storage-openstack-user-infrastructure.adoc
+
+[id="registry-configuring-storage-openstack-user-infra_{context}"]
+= Registry storage for OpenStack with user-provisioned infrastructure
+
+You must set up the storage medium manually and configure the settings in the
+registry CRD.
+
+.Prerequisites
+
+* A cluster on OpenStack with user-provisioned infrastructure.
+* To configure registry storage for OpenStack, you need to provide Registry Operator
+cloud credentials.
+* For Swift on OpenStack storage the Secret is expected to contain the two following keys:
```

Comment:
* Suggest comma after "storage" for easier readability
* Secret -> secret
* two following keys -> following two keys

Same for the same sentence later in the PR